CMakeLists.txt

```diff
@@ -153,11 +153,13 @@ configure_file(include/revng/Runtime/commonconstants.h "${CMAKE_BINARY_DIR}/shar
 configure_file(runtime/early-linked.c "${CMAKE_BINARY_DIR}/share/revng/early-linked.c" COPYONLY)
 configure_file(scripts/revng "${CMAKE_BINARY_DIR}/bin/revng")
 configure_file(scripts/revng-merge-dynamic "${CMAKE_BINARY_DIR}/bin/revng-merge-dynamic" COPYONLY)
+file(COPY scripts/revng_merge_dynamic DESTINATION "${CMAKE_BINARY_DIR}/bin")
```

Comment:
Note: this is needed because `orc install --test` is performed before installation, so the package would not be found.
It does not end up in `root/bin` (the `install` directive below copies the module to `lib/python` as agreed)

Reply:
Use `configure_file`, `file(COPY ...)` is run at configure-time and never again.

Reply:
Does this mean that while in the build directory we will use the installed version? Not OK!